src/renderers/shaders/ShaderLib/distanceRGBA_frag.glsl

```diff
@@ -1,4 +1,4 @@
-uniform vec3 lightPos;
+uniform vec4 lightPos;
```

Comment:
We are not at risk of running out of uniforms in this shader, so I am not sure why we are packing cutoffDistance into lightPosition.  Are you viewing it as a homogeneous coordinate?  Sort of but not really.  Thus this packing is potentially confusing as we have to maintain this code over the years.